src/python/turicreate/toolkits/activity_classifier/_mx_model_architecture.py

```diff
@@ -104,9 +104,6 @@ def _lstm_layer(data, num_hidden, seq_len, dropout):
 def _fit_model_mxnet(model, data_iter, valid_iter, max_iterations, num_gpus, verbose):
     from time import time as _time
 
-    model.bind(data_shapes=data_iter.provide_data,
```

Comment:
Let's avoid modifying the MXNet path. If necessary, you can pass the MXNet model to `_fit_model_tf` as part of the initialization of the TF model.

Note that we don't have to follow the same pattern as what was used for MXNet, if it doesn't make as much sense for us. Maybe just a function that takes all the needed inputs, constructs the TensorFlow model class, trains it, then returns the MXNet weights.